### PR TITLE
fix(tasks): fix flow check

### PR DESCRIPTION
**Pull request by alexkrolick**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/activity-feed/task-new/Task.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ class Task extends React.Component<Props> {
                             </span>
                         ) : null}
                     </div>
-                    {shouldShowActions ? (
+                    {currentUserAssignment && shouldShowActions ? (
```
Comment (alexkrolick) [inline src/elements/content-sidebar/activity-feed/task-new/Task.js]: extracting this result makes flow think the currentUserAssignment could be null

**Pull request merged**